neurots/generate/diametrizer.py

```diff
@@ -18,11 +18,21 @@
 import inspect
 
 import numpy as np
+from morphio import SectionType
 
-from neurots.morphio_utils import STR_TO_TYPES
-from neurots.morphio_utils import TYPE_TO_STR
-from neurots.morphio_utils import root_section_filter
-from neurots.morphio_utils import section_filter
+
+def section_filter(neuron, tree_type=None):
+    """Filters all sections according to type."""
+    if tree_type is None:
+        return list(neuron.iter())
+    return [i for i in neuron.iter() if i.type == tree_type]
+
+
+def root_section_filter(neuron, tree_type=None):
+    """Filters root sections according to type."""
+    if tree_type is None:
+        return list(neuron.root_sections)
+    return [i for i in neuron.root_sections if i.type == tree_type]
```

Comment:
Where are these functions used?

Reply:
later in this file, I copied them from morphio_utils.py which had only these 2 functions, only used in this module. So we simplify things a bit.

Reply:
Ah it was moved from `neurots/morphio_utils.py`, I missed it.